src/main/java/LineSplitter.java

```diff
@@ -0,0 +1,39 @@
+public class LineSplitter {
```

Comment:
I like how you named the class LineSplitter instead of SplittingLines (Using Noun rather than Verb)